docs/pages/pmd/devdocs/pmdtester.md

```diff
@@ -0,0 +1,17 @@
+## Introduction
+Pmdtester is a regression testing tool ensure that new problems and unexpected behaviors will not be introduced to PMD project after fixing an issue , 
```

Comment:
I'd phrase that sentence a bit different, so that sound more fluid (IMHO - I'm also no native speaker...):

Pmdtester is a regression testing tool that ensures no new problems and unexpected behaviors will be introduced to PMD after fixing an issue. It can also be used to verify, that new rules work as expected.